website/content/docs/ui/custom-messages/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+page_title: Manage custom messages
+description: >-
+  Use custom messages in the Vault UI to share system-wide alerts.
+---
+
+# Manage custom messages in the Vault UI
+
+Use custom banners and modals in the Vault UI to share system-wide alerts for all Vault UI users. Users can then dismiss messages by clicking "X" on a banner or "Confirm" for a modal.
+![Vault UI sidebar. The "Custom Messages" button in Settings section is outlined with a red box](/img/custom-messages-sidebar.png)
+
+## Before you start
+
+- **You must have Vault 1.16.0 or higher installed**.
+- **You must have admin-level permission for the Vault UI**.
```

Comment:
Can we define what permissions are needed? When testing this it seems to be https://developer.hashicorp.com/vault/api-docs/system/config-ui.

Reply:
+1 to adding the more granular permissions if we can be specific

Reply:
Good catch! The [api docs for custom messages](https://github.com/hashicorp/vault/pull/24845/files) hasn't been published yet but will link to this page.

@schavis How do we typically define what permissions are needed?

Reply:
This is great! @marcboudreau can you confirm that create and edit require create permissions and edit does not require not update? I think we had a conversation in the past that create and edit are the same thing, but I wanted to be sure.

Reply:
I tested this locally and the policies that are needed are:

```
path "sys/config/ui/custom-messages" {
    capabilities = ["list", "create"]
}

path "sys/config/ui/custom-messages/*" {
    capabilities = ["read"]
}
```